pkg/server/datastore/sqlstore/sqlstore.go

```diff
@@ -3219,6 +3219,10 @@ func pruneRegistrationEntries(tx *gorm.DB, expiresBefore time.Time) error {
 		if err := deleteRegistrationEntrySupport(tx, entry); err != nil {
 			return err
 		}
+		ds.log.WithFields(logrus.Fields{
+			telemetry.SPIFFEID: entry.SpiffeID,
```

Comment:
Do we also want to log the entry id? That will be needed to identify the exact entry that was pruned.

Reply:
Sure, I can add that, thanks!